src/js/start.js

```diff
@@ -85,7 +85,9 @@ $(function () {
     $(this).attr('tabindex', '-1');
   });
 
-  toggleFormInput($('.usa-show_password'), 'Show Password', 'Hide Password');
-  toggleFormInput($('.usa-show_multipassword'), 'Show my typing', 'Hide my typing');
-  validator($('.js-validate_password'));
+  if (/form-controls/.test(window.location.pathname)) {
```

Comment:
The check is to prevent the code from instantiating those plugins if the DOM elements aren't on the page. We could definitely just check individually for the existence of those elements.

I used URLs because it was faster than checking for each element individually, and because I didnt consider that other users might be installing the docs :(